api/src/main/java/org/pmiops/workbench/utils/mappers/EgressEventMapper.java

```diff
@@ -18,5 +24,23 @@ public interface EgressEventMapper {
 
   EgressEventStatus toApiStatus(DbEgressEventStatus status);
 
+  @AfterMapping
+  default void addWindowTiming(DbEgressEvent source, @MappingTarget EgressEvent target) {
+    SumologicEgressEvent sumoEvent = toSumoLogicEvent(source);
+    if (sumoEvent == null) {
+      return;
+    }
+
+    Long timeWindowStart = sumoEvent.getTimeWindowStart();
```

Comment:
Since this code will crash anyways if these are null, I would just use the primitive type instead

Reply:
yep